htdocs/web_portal/views/fragments/serviceTypeInfo.php

```diff
@@ -0,0 +1,23 @@
+<table>
```

Comment:
Could you indent this HTML to improve readability?

Reply:
Included in rebased code.